lib/DeviceManagerPlugin.ts

```diff
@@ -82,6 +81,7 @@ export type DeviceManagerConfig = {
 
 export class DeviceManagerPlugin extends Plugin {
   public config: DeviceManagerConfig;
+  private decoderQueue: DecoderQueue[] = [];
```

Comment:
This property name and type name are misleading since a [Queue](https://www.wikiwand.com/en/FIFO_(computing_and_electronics) is a type of data structure but here you just use this as a simple array